clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/model/ManifestBasedModel.java

```diff
@@ -66,6 +74,19 @@ public String getAccount() {
     return getAccountName();
   }
 
+  public Long getCreatedTime() {
+    Map<String, String> metadata = (Map<String, String>) getManifest().getOrDefault("metadata", new HashMap<>());
+    String timestamp = metadata.get("creationTimestamp");
+    try {
+      return StringUtils.isNotEmpty(timestamp) ?
+          (new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX").parse(timestamp)).getTime() :
+          null;
+    } catch (ParseException e) {
+      log.warn("Failed to parse timestamp: ", e);
+      return null;
+    }
```

Comment:
Minor style nit:  How about de-duping the `return null` and dropping the ternary?

```java
  try {
    if (StringUtils.isNotEmpty(timestamp)) {
      return new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ssX").parse(timestamp)).getTime());
    }
  } catch (ParseException e) {
    log.warn("Failed to parse timestamp: ", e);
  }
  return null;
```

Reply:
SGTM